### DISCUSSION TITLE
HELP  Firebase Cloud MessagingPrivate Key JSON

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6269] [FAQs]
PLease HElP how to optain the Firebase Cloud MessagingPrivate Key JSON   , lost in which one , also how to reay the output of it
thanks , also it possible to use pusher , anyone have a webhook template so i can try it plz thanks

**Comment by brixxnatt** [comment]
anyone ?

**Comment by lastpeony** [comment]
You can find your cloud messaging api key through firebase console.
